packages/vscode-extension/src/loggers/VSConsoleLogger.ts

```diff
@@ -35,7 +40,7 @@ export default class VSConsoleLogger extends ConsoleLogger {
 			output.appendLine(this.getExecutionDetails());
 			this._addExecutionDetailsToLog = false;
 		}
-
+		this.checkForCorruptedJar(message);
```

Comment:
Why are we adding the checkForCorruptedJar in the logger? 
Cannot we move it to another class more suitable, and decouple the logic for logging and checking for corrupted jar? 
they can be call separately.

Reply:
I agree with you, but I didn't find a better way.